app/uk/gov/hmrc/gform/gform/FormService.scala

```diff
@@ -31,4 +34,25 @@ class FormService {
         FormComponentValidation(formComponent, FieldOk(fv, cv.replaceAll(",", "")))
       case _ => formComponentValidation
     }
+
+  def splitFormComponentValidation(
+    optionFcv: Option[(FormComponent, FormFieldValidationResult)]): Option[FormComponentValidation] =
+    optionFcv.map(fcv => FormComponentValidation(fcv._1, fcv._2))
```

Comment:
._1 & ._2 ??? Do you think we could a bit better with naming? If you don't want to replace the tuple with a case class you may do a partial function